runhouse/main.py

```diff
@@ -679,6 +311,103 @@ def cluster_list(
         print_sky_clusters_msg(len(sky_clusters))
 
 
+@cluster_app.command("keep-warm")
+def cluster_keep_warm(
+    cluster_name: str,
+    mins: Optional[int] = typer.Option(
+        -1,
+        help="Amount of time (in min) to keep the cluster warm after inactivity. If set to -1, keeps cluster warm indefinitely.",
+    ),
+):
+    """Keep the cluster warm for given number of minutes after inactivity.
+
+    Example:
+        ``$ runhouse cluster keep-warm /sashab/rh-basic-cpu``
+
+    """
+
+    current_cluster = get_cluster_or_local(cluster_name)
+
+    try:
+        if not current_cluster.is_up():
+            print_bring_cluster_up_msg(cluster_name=cluster_name)
+            return
+        current_cluster.keep_warm(mins=mins)
+    except ValueError:
+        console.print(f"{cluster_name} is not saved in den.")
```

Comment:
dumb nit - we should be consistent in how we write it
```suggestion
        console.print(f"{cluster_name} is not saved in Den.")
```